### PR TITLE
replication to local disk storage

### PR DESCRIPTION
**Pull request by kmlebedev**

### CODE REVIEW

**Comment by kmlebedev** [comment]
@chrislusf There is an understanding of what these messages are ?
```
replicate_1  | 2021/01/26 17:49:52 A pubsub.Message was never Acked or Nacked (/Users/kmlebedev/GolandProjects/seaweedfs/weed/replication/sub/notification_gocdk_pub_sub.go:42)
replicate_1  | 2021/01/26 17:49:52 A pubsub.Message was never Acked or Nacked (/Users/kmlebedev/GolandProjects/seaweedfs/weed/replication/sub/notification_gocdk_pub_sub.go:42)
```

**Comment by chrislusf** [comment]
> ```
> replicate_1  | 2021/01/26 17:49:52 A pubsub.Message was never Acked or Nacked (/Users/kmlebedev/GolandProjects/seaweedfs/weed/replication/sub/notification_gocdk_pub_sub.go:42)
> replicate_1  | 2021/01/26 17:49:52 A pubsub.Message was never Acked or Nacked (/Users/kmlebedev/GolandProjects/seaweedfs/weed/replication/sub/notification_gocdk_pub_sub.go:42)
> ```

Need to add Ack() the message in filer_replication.go.

**Comment by chrislusf** [comment]
I added code to handle this https://github.com/chrislusf/seaweedfs/commit/ad2a20c8a5827461d07ab2a46932d0c1c89d1b37

File: weed/replication/sink/localsink/local_sink.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+	return nil
+}
+
+func (localsink *LocalSink) UpdateEntry(key string, oldEntry *filer_pb.Entry, newParentPath string, newEntry *filer_pb.Entry, deleteIncludeChunks bool, signatures []int32) (foundExistingEntry bool, err error) {
+	if localsink.isMultiPartEntry(key) {
+		return true, nil
+	}
+	glog.V(4).Infof("Update Entry key: %s", key)
+	// do delete and create
```
Comment (chrislusf) [inline weed/replication/sink/localsink/local_sink.go:99]: still work in progress, right?

Comment (kmlebedev) [reply weed/replication/sink/localsink/local_sink.go:99]: No, work  is done
if foundExistingEntry is false replicator calls DeleteEntry then CreateEntry

**Comment by kmlebedev** [comment]
> I added code to handle this [ad2a20c](https://github.com/chrislusf/seaweedfs/commit/ad2a20c8a5827461d07ab2a46932d0c1c89d1b37)

Thank you!

**Pull request merged**